nim-mode.el

```diff
@@ -924,6 +936,11 @@ You don't need to set this if the nim executable is inside your PATH."
   :type '(repeat string)
   :group 'nim)
 
+;; FIXME
+(defun nim-get-project-root ()
```

Comment:
Implement it with help of `(nim-find-project-main-file)`. Feel free to rename the function too.